src/SFA.DAS.EmploymentCheck.TokenServiceStub/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using SFA.DAS.EmploymentCheck.TokenServiceStub.Configuration;
+using SFA.DAS.EmploymentCheck.TokenServiceStub.Http;
+using SFA.DAS.EmploymentCheck.TokenServiceStub.Services;
+using SFA.DAS.TokenService.Api.Client;
+
+namespace SFA.DAS.EmploymentCheck.TokenServiceStub
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection AddTokenServiceStubServices(this IServiceCollection services)
+        {
+            services.AddSingleton<ITokenServiceApiClient, TokenServiceApiClientStub>();
```

Comment:
Does this TokenServiceApiClientStub need removing?

Reply:
no because it's used in all environments except PROD.

question is whether we can/should use the real token service in PRE-PROD @notquitecraig  @DavidBSteele  ?